test/test-poll-state-cleanup.c

```diff
@@ -0,0 +1,42 @@
+#include <assert.h>
+#include "uvwasi.h"
+
+int main(void) {
+  uvwasi_t uvwasi;
+  uvwasi_options_t init_options;
+  uvwasi_subscription_t sub;
+  uvwasi_event_t event;
+  uvwasi_size_t nevents;
+  uvwasi_errno_t err;
+
+  init_options.in = 0;
+  init_options.out = 1;
+  init_options.err = 2;
+  init_options.fd_table_size = 3;
+  init_options.argc = 0;
+  init_options.argv = NULL;
+  init_options.envp = NULL;
+  init_options.preopenc = 0;
+  init_options.preopens = NULL;
+  init_options.allocator = NULL;
```

Comment:
I wonder if its worth adding a `uvwasi_options_init()` helper version to avoid writing out all those null initializes?  Similar to phtread_attr_init:  http://man7.org/linux/man-pages/man3/pthread_attr_init.3.html?

Reply:
I like the idea: https://github.com/cjihrig/uvwasi/pull/141